### PR TITLE
Php8v2composer update

### PR DESCRIPTION
**Pull request by joesken**
Removed the second shared:folders
Updated version of Deployer

Was going to fix the naming of the branch but you know what, if it ain't broke....

### CODE REVIEW

**Comment by joesken** [comment]
Including everybody so they are aware of the change before I merge it in. Awaiting approval.

**Review by jasmine-aonach — APPROVED** [review]
looks good

**Review by enriquetarazona — APPROVED** [review]
looks good

**Pull request merged**
